### PR TITLE
Python fuzzing fixes - disable leak checking, remove flags in atheris.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: infra/base-images/base-builder/Dockerfile [hunk 1]
```diff
@@ -57,11 +57,9 @@ RUN export PYTHON_DEPS="\
     rm -r /tmp/Python-$PYTHON_VERSION.tar.xz /tmp/Python-$PYTHON_VERSION && \
     apt-get remove -y $PYTHON_DEPS  # https://github.com/google/oss-fuzz/issues/3888
 
-# Install latest atheris for python fuzzing.
-RUN pip3 install atheris
-
-# Install pyinstaller for python fuzzer packaging.
-RUN pip3 install pyinstaller==4.1
+# Install latest atheris for python fuzzing, pyinstaller for fuzzer packaging.
+RUN export CFLAGS="" CXXFLAGS="" && \
```
Comment (mbarbella-chromium) [inline infra/base-images/base-builder/Dockerfile:61]: Any reason this needs to be done with export? Could this cause issues with any later commands?

**Review by mbarbella-chromium — APPROVED** [review]

**Comment by jonathanmetzman** [comment]
Can you pull again and trigger CI.
I want CI to test this using https://github.com/google/oss-fuzz/pull/4679

**Comment by inferno-chromium** [comment]
> Can you pull again and trigger CI before landing this please.
> I want CI to test this using #4679

Yes did it.

File: infra/base-images/base-builder/Dockerfile (hunk 1 above)
Comment (inferno-chromium) [reply infra/base-images/base-builder/Dockerfile]: Yes removed, using directly.

**Pull request merged**